server/user/jwt.go

```diff
@@ -1,27 +1,31 @@
-package middleware
+package user
```

Comment:
I like this a lot more, this is a much better package organization. (I gotta say some nice stuff instead of just picking at packages!)